.github/workflows/build-and-publish.yaml

```diff
@@ -0,0 +1,23 @@
+name: Build and Publish PyPi Packages
+run-name: ${{ format('Release {0}', inputs.release_type)}} Package by @${{ github.actor }}
+on:
+  workflow_dispatch:
+    inputs:
+      release_type:
+        type: choice
+        description: The type of release
+        options:
+        - Major
+        - Minor
+        - Patch
+      publish_vulnerabilities:
+        type: string
+        default: true
+
+jobs:
+  build-and-pubish:
+    name: Build and publish Python packages to PyPi
+    uses: IABTechLab/uid2-shared-actions/.github/workflows/shared-publish-to-pypi-versioned.yaml@kcc-UID2-2330-create-workflow-for-python
```

Comment:
Should you not also pass through the publish_vulnerability value?

Reply:
Added